WordPress/gutenberg
# ComboboxControl / FormTokenField: Add flag for larger default size

Part of #39397

## What?

Adds the temporary prop `__next36pxDefaultSize` to opt into the new 36px height default size.

## Why?

As part of the effort to move toward consistent component sizes.

This PR had to include two components at once because `ComboboxControl` depends on some `FormTokenField` subcomponents internally.

## How?

See inline comments for notable points.

## Testing Instructions

1. `npm run storybook:dev`
2. See stories for `ComboboxControl` and `FormTokenField`. They should look nice with the `__next36pxDefaultSize` prop enabled. With the prop disabled, it should look the same as in trunk ([ComboboxControl](https://wordpress.github.io/gutenberg/?path=/story/components-comboboxcontrol--default), [FormTokenField](https://wordpress.github.io/gutenberg/?path=/story/components-formtokenfield--default)).
3. For `FormTokenField`, also test the case where there are a lot of tokens and it wraps to the next line.

## Known issues

I imagine we'll want to make the paddings in the suggestion list roomier to match the larger input size, but I'll leave that to a future PR.
